packages/stark-testing/karma.conf.ci.js

```diff
@@ -43,7 +43,8 @@ const rawKarmaConfig = {
 			],
 			resolve: {},
 			transforms: [
-				require("karma-typescript-angular2-transform") // see https://github.com/monounity/karma-typescript-angular2-transform
+				require("karma-typescript-angular2-transform"), // see https://github.com/monounity/karma-typescript-angular2-transform
+				require("karma-typescript-es6-transform")() // add preset in a way that subpackages can find it
```

Comment:
Can you also add the link to this package? https://github.com/monounity/karma-typescript/tree/master/packages/karma-typescript-es6-transform